src/pages/_app.tsx

```diff
@@ -33,12 +33,15 @@ type Props = AppProps & {
   hostname: {
     origin: string
   }
+  pageProps: Record<string, never>
```

Comment:
Question: Is this how you make the prop "required"?

Reply:
It doesn't make it required, but makes TypeScript happy since we're now accessing the pageProps as an object in this file, instead of just passing it through to getLayout on line 176.